html/user/account_ownership.php

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env php
```

Comment:
remove this line. It gets output to the user.  It is only required if you are going to run the PHP from the command line.

Reply:
Does it not count as CLI when running as part of makeproject? I'll remove it and make a separate 'overwrite keys' op file (which can be run from CLI, not from web/makeproject).

Reply:
This file - account_ownership.php is only executed by the web application.  So it doesn't need the line.

The file generate_account_ownership_keys.php also as this bit of code, but it needs it because it is run as CLI.  I'm ok with the line remaining remaining in generate_account_ownership_keys.php because of that CLI usage (and because it is only used on the webapp in the ops pages)

Reply:
Removed from the check op script, kept it in the generate op script.